db/migrations/20221229101326-create-user.js

```diff
@@ -32,6 +32,10 @@ module.exports = {
         type: Sequelize.STRING,
         allowNull: false
       },
+      role: {
+        type: Sequelize.STRING,
+        defaultValue: 'user'
+      },
```

Comment:
This migration has already been ran in production, you can edit it here. You need to create a new migration that just adds this column.